translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+          "message": [
+              "Ez az esemény már ki van kapcsolva."
+          ]
+      },
+      "22": {
+          "note": "Felemelve amikor egy érvénytelen időzítőt állítanál be. (skyblock events config)",
+          "message": [
+              "**{time}** egy érvénytelen idő. Csak **0**, **5**, **10**, vagy **30** lehetséges."
+          ]
+      },
+      "23": {
+          "note": "Felemelve, amikor egy figyelmeztetés már be van kapcsolva. (skyblock events config)",
+          "message": [
+              "Ez a figyelmeztetés már be van kapcsolva."
+          ]
+      },
+      "24": {
+          "note": "Felemelve, amikor egy figyelmeztetés már ki van kapcsolva. (skyblock events config)",
+          "message": [
+              "Ez a figyelmeztetés már ki van kapcsolva."
+          ]
+      },
+      "25": {
+          "note": "Felemelve, amikor egy játékosnak nincsenek lehozandó SkyBlock profiljai.",
+          "message": [
+              "**{name}**-nek nincsenek SkyBlock profiljai."
+          ]
+      }
+    },
+    "checkFailure": {
+      "0": {
+        "note": "Felemelve, amikor egy felhasználónak nincs prémiumja, de szükséges.",
+        "message": [
+          "A parancs használatához rendelkezned kell a SkyKings Premium-mal.\n",
+          "Több információért használd a `/premium info` parancsot."
+        ]
+      },
+      "1": {
+        "note": "Felemelve, amikor egy esemény már fut.",
+        "message": [
+          "Már fut egy esemény ezen a szerveren."
+        ]
+      },
+      "2": {
+        "note": "Felemelve, amikor egy esemény nem fut.",
+        "message": [
+          "Nem fut esemény ezen a szereveren."
+        ]
+      },
+      "3": {
+        "note": "Felemelve, amikor a parancsidéző nem ellenőrzött.",
+        "message": [
+          "Ellenőrzöttnek kell lenned, hogy használni tudd ezt a parancsot.\n",
+          "Ellenőrizhetsz a `/verify` paranccsal."
+        ]
+      },
+      "4": {
+        "note": "Felemelve, amikor a parancsidéző már ellenőrzött.",
+        "message": [
+          "Már ellenőrizve vagy."
+        ]
+      },
+      "5": {
+        "note": "Felemelve, amikor a parancsidéző már az eseményben van.",
+        "message": [
+          "Már az eseményben vagy."
+        ]
+      },
+      "6": {
+        "note": "Felemelve, amikor a parancsiddéző nincs az eseményben.",
+        "message": [
+          "Nem vagy az eseményben. Csatlakozz az `/event join` paranccsal."
+        ]
+      },
+      "7": {
+        "note": "Felemelve, amikor a parancsidézőnek nincsenek engedélyei.",
+        "message": [
+          "Nincsenek engedélyeid, hogy használd ezt a parancsot.\n"
+        ]
+      },
+      "8": {
+        "note": "Felemelve, amikor egy prémium funkciót prémium nélkül akarsz használni.",
+        "message": [
+          "SkyKings Premium szükséges, hogy végrehajtsd ezt az akciót.\n",
+          "Több információért használd a `/premium info` parancsot."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`